numpy/core/src/umath/ufunc_object.c

```diff
@@ -2175,14 +2175,13 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
                 if (n != num_dims) {
                     if (num_dims == 1) {
                         PyErr_Format(PyExc_ValueError,
-                                     "axis list element %d should be a tuple "
-                                     "with a single elements, an integer, or "
-                                     "None", i);
+                                     "axis item %d should be a list with a "
+                                     "single element, an integer, or None", i);
                     }
                     else {
                         PyErr_Format(PyExc_ValueError,
-                                     "axis list element %d should be a tuple "
-                                     "with %d elements or None", i, num_dims);
+                                     "axis item %d should be a list with %d "
+                                     "elements or None", i, num_dims);
```

Comment:
This can be mistakenly read as "a list with no elements". A comma would help, or so would swapping the order